### PR TITLE
tanka v0.9.2 (new formula)

### PR DESCRIPTION
**Pull request by tombrk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This adds [Tanka](https://tanka.dev), a Kubernetes configuration management tool build at Grafana Labs.

### CODE REVIEW

**Review by claui — CHANGES_REQUESTED** [review]
Can you improve the test so it exercises at least a tiny bit of `tk`’s functionality?

**Comment by sh0rez** [comment]
@claui Sure! I've added a test that uses https://github.com/sh0rez/grafana.libsonnet (an example library for Tanka) to render some YAML. This is the advanced operation Tanka does without requiring user input, but definitely checks for functionality, such as the integrated Jsonnet compiler and the processing stages.

File: Formula/tanka.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Tanka < Formula
+  desc "Flexible, reusable and concise configuration for Kubernetes using Jsonnet"
+  homepage "https://tanka.dev"
+  url "https://github.com/grafana/tanka.git",
+    :tag => "v0.9.2"
```
Comment (SMillerDev) [inline Formula/tanka.rb:5]: Please specify a commit hash here as well.

Comment (sh0rez) [reply Formula/tanka.rb]: Done, PTAL

**Comment by SMillerDev** [comment]
Can you squash your commits?

**Comment by sh0rez** [comment]
@smillerdev done!

**Comment by SMillerDev** [comment]
Thanks @sh0rez  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**